registry/quilt_server/const.py

```diff
@@ -9,7 +9,8 @@
 
 PUBLIC = 'public' # This username is blocked by Quilt signup
 VALID_NAME_RE = re.compile(r'^[a-zA-Z]\w*$')
-EMAILREGEX = re.compile(r'^([^\s@]+)@([^\s@]+)$')
+VALID_USERNAME_RE = re.compile(r"^[^\d\W]\w*\Z", re.UNICODE)
```

Comment:
We should be using `VALID_NAME_RE` for usernames. If we currently allow them to start with `_`, then that's a bug.